checkout-core/src/main/java/com/adyen/checkout/core/api/HttpClientExt.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 11/4/2022.
+ */
+
+package com.adyen.checkout.core.api
+
+import com.adyen.checkout.core.log.LogUtil
+import com.adyen.checkout.core.log.Logger
+import com.adyen.checkout.core.model.ModelObject
+import com.adyen.checkout.core.model.toStringPretty
+import org.json.JSONObject
+
+private val TAG = LogUtil.getTag()
+
+fun <T : ModelObject> HttpClient.get(
+    path: String,
+    responseSerializer: ModelObject.Serializer<T>
+): T {
+    Logger.v(TAG, "GET - $path")
```

Comment:
One last comment 🙈  Maybe we can make the path logs as `Logger.d` and keep the request and response bodies as `Logger.v`?